apps/condo/domains/ticket/access/UserFavoriteTicket.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Generated by `createschema ticket.UserFavoriteTicket 'user:Relationship:User:CASCADE; ticket:Relationship:Ticket:CASCADE;'`
+ */
+
+const get = require('lodash/get')
+const isEmpty = require('lodash/isEmpty')
+const omit = require('lodash/omit')
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { getById, getByCondition } = require('@open-condo/keystone/schema')
+
+const { queryOrganizationEmployeeFor, queryOrganizationEmployeeFromRelatedOrganizationFor } = require('@condo/domains/organization/utils/accessSchema')
+
+async function canReadUserFavoriteTickets ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.isAdmin || user.isSupport) return {}
+
+    return {
+        user: { id: user.id },
+        organization: {
+            OR: [
+                queryOrganizationEmployeeFor(user.id),
+                queryOrganizationEmployeeFromRelatedOrganizationFor(user.id),
+            ],
+        },
+    }
+}
+
+async function canManageUserFavoriteTickets ({ authentication: { item: user }, originalInput, operation, itemId }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return true
+
+    let organizationId
+
+    if (operation === 'create') {
+        const ticketId = get(originalInput, ['ticket', 'connect', 'id'])
+        const userId = get(originalInput, ['user', 'connect', 'id'])
+        if (user.id !== userId) {
+            return false
+        }
+
+        const ticket = await getById('Ticket', ticketId)
+        organizationId = ticket.organization
+    } else if (operation === 'update') {
+        if (!isEmpty(omit(originalInput, ['deletedAt', 'dv', 'sender']))) {
+            return false
+        }
+
+        const userFavoriteTicket = await getById('UserFavoriteTicket', itemId)
+        if (userFavoriteTicket.user !== user.id) {
+            return false
+        }
+
+        organizationId = userFavoriteTicket.organization
+    }
+
+    const organizationEmployee = await getByCondition('OrganizationEmployee', {
+        organization: {
+            id: organizationId,
+            OR: [
+                queryOrganizationEmployeeFor(user.id),
+                queryOrganizationEmployeeFromRelatedOrganizationFor(user.id),
+            ],
+        },
+        user: { id: user.id },
+        deletedAt: null,
```

Comment:
perhaps a check for blocking should be added here?

```suggestion
        deletedAt: null,
        isBlocked: false,
```